modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/QueryTransactionWrapperSelfTest.java

```diff
@@ -49,15 +47,6 @@ public class QueryTransactionWrapperSelfTest extends BaseIgniteAbstractTest {
     @Mock
     private IgniteTransactions transactions;
 
-    @Test
-    public void throwsExceptionForDdlWithExternalTransaction() {
```

Comment:
because the validation was moved to another place (see  validateParsedStatement() ). Existing test are covered the case ( see https://github.com/apache/ignite-3/pull/2740/files#diff-15adebe06eeadc5173fe83a19d4991da5e11d51e6ade992353627b5f4760e08cR145)